From c4f2a483dcc5920871f8534b0ce731b6a33a1516 Mon Sep 17 00:00:00 2001
From: quni <quni.inne@gmail.com>
Date: Fri, 24 May 2024 13:07:58 +0200
Subject: [PATCH 5/6] add: loop prop to hook

---
 src/hooks/useInViewTrailAnimation.tsx | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/src/hooks/useInViewTrailAnimation.tsx b/src/hooks/useInViewTrailAnimation.tsx
index d87ac6c..c3702b5 100644
--- a/src/hooks/useInViewTrailAnimation.tsx
+++ b/src/hooks/useInViewTrailAnimation.tsx
@@ -1,6 +1,10 @@
 import { useTrail, useInView } from "@react-spring/web";
 
-function useInViewTrailAnimation(items: string[], duration = 300) {
+function useInViewTrailAnimation(
+  items: string[],
+  duration = 300,
+  loop: boolean,
+) {
   const [inViewRef, inView] = useInView({});
 
   const trail = useTrail(items.length, {
@@ -9,8 +13,7 @@ function useInViewTrailAnimation(items: string[], duration = 300) {
       opacity: inView ? 1 : 0,
       transform: inView ? "translate3d(0,0px,0)" : "translate3d(0,40px,0)",
     },
-    config: { mass: 1, tension: 2000, friction: 400, duration },
-
+    config: { mass: 1, tension: 2000, friction: 400, duration, loop },
     reset: true,
   });
 
-- 
2.45.0

