From c2f21ec39c0346861a07f6714276a188f75a7b80 Mon Sep 17 00:00:00 2001
From: quni <quni.inne@gmail.com>
Date: Fri, 24 May 2024 13:08:16 +0200
Subject: [PATCH 6/6] style: styling imports

---
 src/main.tsx | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/main.tsx b/src/main.tsx
index 190f0d1..61c6528 100644
--- a/src/main.tsx
+++ b/src/main.tsx
@@ -1,10 +1,12 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-import { injectSpeedInsights } from "@vercel/speed-insights";
-import { inject } from "@vercel/analytics";
 import App from "./App.tsx";
 import "./styles/main.css";
 
+// Analytics
+import { injectSpeedInsights } from "@vercel/speed-insights";
+import { inject } from "@vercel/analytics";
+
 // Vercel insights
 injectSpeedInsights();
 inject();
-- 
2.45.0

